test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ OPTIONAL ] As part of the [Project Requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you were asked to  add 

> A test.md file with the text "This file should be ignored by git".
> Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
But I noticed you didn't fulfill this requirement. Please note that this project was to help you learn how to make use of some of these files in this case the `.gitignore` so you can make good use of it in your future projects :100: 
- Even though this is an optional comment, I will suggest you do it to serve as practice for you :+1: 
- [ ] You can read more about how `.gitignore` works [here](https://www.bmc.com/blogs/gitignore/).